sites/all/themes/brin/template.php

```diff
@@ -459,9 +459,14 @@ function brin_ting_object_cover($variables) {
  */
 function brin_preprocess_ting_search_carousel(&$vars) {
   if (isset($vars['settings'])) {
-    $vars['settings']['slidesToShow'] = 6;
+    $vars['settings']['slidesToShow'] = 5;
```

Comment:
.. men @xendk hvad siger du , skal vi lade det blive ved 6 . btw på ereolen for børn er det også 6